results.tex

```diff
@@ -0,0 +1,143 @@
+\section{Results}
+
+\subsection{Past-Predictable Future Scenario}
+
+\ref{tab:sim_result} lists the important metrics
+obtained from the first simulation. The following
+values are the EU inventory and history at year 2050.
+
+\begin{table}[h]
+	\centering
+	\label{tab:sim_result}
+	\caption {Simulation Results}
```

Comment:
I slightly prefer table captions to come *after* the table anyway. As it currently stands, the paper is inconsistent about this. Some come after. Some come before.